bookshelf/README.md

```diff
@@ -2,5 +2,13 @@
 A book sharing aplication developed in ROR
 
 ## Project Setup
-1. Clone the repository from..
+1. Clone the repository in your directory by running this command
+   git clone https://github.com/Talentica/TeamPolyglot-ROR.git
+2. Change the directory to bookshelf
+    $ cd bookshelf
+3. Create the database 
```

Comment:
Don't we need to install dependencies by running bundler. And perhaps instal the bundler gem as well if it's first time.